OpenRA.Mods.Common/Scripting/Global/BeaconGlobal.cs

```diff
@@ -27,9 +27,9 @@ public BeaconGlobal(ScriptContext context) : base(context)
 
 		[Desc("Creates a new beacon that stays for the specified time at the specified WPos. " +
 			"Does not remove player set beacons, nor gets removed by placing them.")]
-		public Beacon New(Player owner, WPos position, int duration = 30 * 25, bool showRadarPings = true, string palettePrefix = "player")
+		public Beacon New(Player owner, WPos position, int duration = 30 * 25, bool showRadarPings = true)
 		{
-			var playerBeacon = new Beacon(owner, position, duration, palettePrefix);
+			var playerBeacon = new Beacon(owner, position, duration, "player", true, "beacon", "arrow", "circle");
```

Comment:
I'm not sure if that's the best way. How about adding default arguments instead?